### ISSUE TITLE
[Firefox] Not compatible with multiprocess

### ISSUE THREAD

**Issue by f6fvy** [NicolaeNMV/BehindTheOverlay#20]
The title says it all.

It prevents Firefox to run as multiprocess (e10s / Electrolysis).

**Comment by B00ze64** [comment]
I don't think this is valid anymore, I see a dozen FF processes on FF77 with this WX installed...
